modules/ROOT/pages/mobile.adoc

```diff
@@ -49,30 +49,31 @@ tinymce.init({
 });
 ----
 
+[[mobiledefaultsforselectedsettings]]
 === Mobile defaults for selected settings
 
 These mobile-specific default values have been set to disable unsupported settings for mobile devices or to provide the best experience without configuration from developers.
 
 The following settings have mobile-specific default values:
 
-* link:{baseurl}/configure/editor-appearance/#menubar[`menubar`] - defaults to `false` on mobile phones.
-* link:{baseurl}/configure/editor-appearance/#toolbar_mode[`toolbar_mode`] - defaults to `scrolling` on mobile devices. The toolbar will <<sidescrollingtoolbarsonmobile,side-scroll by default>>.
-* link:{baseurl}/configure/editor-appearance/#toolbar_sticky[`toolbar_sticky`] - Sticky Toolbar is not supported on mobile devices and defaults to `false`.
-* link:{baseurl}/plugins/opensource/table/#table_grid[`table_grid`] - Table grid is not supported on mobile devices and defaults to `false`. When creating tables on mobile, a dialog is shown instead of the table grid.
-* link:{baseurl}/configure/editor-appearance/#resize[`resize`] - Resizing is not supported on mobile devices and defaults to `false`.
-* link:{baseurl}/configure/advanced-editing-behavior/#object_resizing[`object_resizing`] - Object resizing is not supported on mobile devices and defaults to `false`.
+* xref:configure/editor-appearance.adoc#menubar[`menubar`] - defaults to `false` on mobile phones.
+* xref:configure/editor-appearance.adoc#toolbar_mode[`toolbar_mode`] - defaults to `scrolling` on mobile devices. The toolbar will xref:sidescrollingtoolbarsonmobile[side-scroll by default].
+* xref:configure/editor-appearance.adoc#toolbar_sticky[`toolbar_sticky`] - Sticky Toolbar is not supported on mobile devices and defaults to `false`.
+* xref:plugins/opensource/table.adoc#table_grid[`table_grid`] - Table grid is not supported on mobile devices and defaults to `false`. When creating tables on mobile, a dialog is shown instead of the table grid.
```

Comment:
This xref doesn't resolve as the `table_grid` id is missing. It looks like it's missing in the `table_grid.adoc` page.

- [ ] Fix broken xref

Reply:
Done! I added the missing id in table_grid.adoc